lusSTR/tests/test_suite.py

```diff
@@ -63,8 +63,6 @@ def test_flank_anno():
         lusSTR.annot.main(args)
         outfile_name = os.path.splitext(outfile.name)[0]
         outfile_name_output = f'{outfile_name}_flanks_anno.txt'
-        import subprocess
-        subprocess.check_call(['cp', outfile_name_output, 'FLANKY'])
```

Comment:
Ooops! 😅 I'm pretty sure I dropped this in here at some point for debugging purposes.